Core.DomainModel/GDPR/DataProcessingAgreement.cs

```diff
@@ -9,17 +9,18 @@
 
 namespace Core.DomainModel.GDPR
 {
-    public class DataProcessingAgreement : 
-        HasRightsEntity<DataProcessingAgreement, DataProcessingAgreementRight, DataProcessingAgreementRole>, 
-        IHasName, 
-        IOwnedByOrganization, 
+    public class DataProcessingAgreement :
+        HasRightsEntity<DataProcessingAgreement, DataProcessingAgreementRight, DataProcessingAgreementRole>,
```

Comment:
newline for alle?

Reply:
ja for hvert interface :-)... det er nemmere at læse det så